### ISSUE TITLE
Validate snippets content

### ISSUE THREAD

**Issue by leandrocp** [BeaconCMS/beacon#433]
Adding an invalid snippet in a page, for eg in meta tags cause the page to fail at runtime due to the lazy evaluation done only after requesting a page.

We should better validate the content before saving any resource that contains snippets.

**Comment by leandrocp** [comment]
Duplicate of https://github.com/BeaconCMS/beacon/issues/450
